api/src/main/resources/workbench.yaml

```diff
@@ -648,9 +648,27 @@ definitions:
   SearchRequest:
```

Comment:
Use example: in here?

(It's hard to tell what these mean as it stands.)

Reply:
I agree.  I put in "typical" and "minimal" examples, although they may change a bit as the rest of the crit gets built out.